run_linux.go

```diff
@@ -479,6 +479,15 @@ func (b *Builder) addNetworkConfig(rdir, hostPath string, chownOpts *idtools.IDP
 	if len(dnsSearch) > 0 {
 		search = dnsSearch
 	}
+
+	if b.Isolation == IsolationOCIRootless {
+		ns := namespaceOptions.Find(string(specs.NetworkNamespace))
+		if ns != nil && !ns.Host && ns.Path == "" {
+			// if we are using slirp4netns, also add the built-in DNS server.
+			nameservers = append([]string{"10.0.2.3"}, nameservers...)
```

Comment:
Can you at least log that we are doing this.  Is it possible to add the comment line here as well like you did in podman?

Reply:
I've added a debug log message